gradle.properties

```diff
@@ -20,4 +20,4 @@ org.gradle.dependency.verification.console=verbose
 VERSION_MAJOR=1
 VERSION_MINOR=8
 VERSION_PATCH=0
-VERSION_BUILD=3
+VERSION_BUILD=5
```

Comment:
could set this to 1, then we have it directly available :)

Reply:
but please not in the merge commit, that would be evil